src/main/java/no/sikt/sws/SnapshotTakingLambda.java

```diff
@@ -0,0 +1,52 @@
+package no.sikt.sws;
+
+import com.amazonaws.http.HttpMethodName;
+import com.amazonaws.services.lambda.runtime.Context;
+import no.sikt.sws.exception.SearchException;
+import nva.commons.apigateway.ApiGatewayHandler;
+import nva.commons.apigateway.RequestInfo;
+import nva.commons.apigateway.exceptions.ApiGatewayException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.sql.Timestamp;
+
+import static no.sikt.sws.constants.ApplicationConstants.BACKUP_BUCKET_NAME;
+
+public class SnapshotTakingLambda extends ApiGatewayHandler<Void, String> {
+
+    private static final Logger logger = LoggerFactory.getLogger(RegisterSnapshotRepoHandler.class);
+    public OpenSearchClient openSearchClient = new OpenSearchClient();
+
+    public SnapshotTakingLambda() {
+        super(Void.class);
+    }
+
+    @Override
+    protected String processInput(Void input, RequestInfo requestInfo, Context context) throws ApiGatewayException {
+        var timestamp = new Timestamp(System.currentTimeMillis()).toString();
```

Comment:
Can you perhaps archive the same result by using another import with simpler code? 
The Timestamp you are using is for SQL

Reply:
With Tests it would be easier to understand what the output would look like, and help out with simplifing

Reply:
https://docs.oracle.com/javase/8/docs/api/java/sql/Timestamp.html is widely used not only in sql?
Do you have other timestamp generator options?

Reply:
I am little bit confused how to test/mock properly RequestInfo. Hm.